acceptance/replication_test.go

```diff
@@ -34,17 +34,17 @@ import (
 )
 
 func countRangeReplicas(client *client.DB) (int, error) {
-	r, err := client.Scan(keys.Meta2Prefix, keys.Meta2Prefix.PrefixEnd(), 10)
+	rng, err := client.Scan(keys.Meta2Prefix, keys.Meta2Prefix.PrefixEnd(), 0)
```

Comment:
The number of nodes shouldn't affect this scan. We're looking for the row containing the range descriptor for the first range. That should be a fixed location. In fact, we could replace the scan with a `Get`.